commands/user

```diff
@@ -19,7 +19,7 @@ DBUS_ROOT_PATH="/xyz/openbmc_project/user"
 DBUS_ATTRIBUTES="xyz.openbmc_project.User.Attributes"
 DBUS_DELETE="xyz.openbmc_project.Object.Delete"
 
-USERNAME_REGEX="^[\.a-zA-Z0-9_-]+$"
+USERNAME_REGEX="^[a-zA-Z_][a-zA-Z0-9_]*$"
```

Comment:
It's against RFC but fully according [WebUI](https://github.com/openbmc/webui-vue/blob/f763cd2e39ffce9b10191402243e8704794f08ff/src/views/SecurityAndAccess/UserManagement/ModalUser.vue#L296).

Reply:
RFC is about generic POSIX systems. BMC may impose additional restrictions.

Reply:
It would be nice though to mention the source of those requirements/restrictions in the commit log.